diff --git a/dist/index.d.ts b/dist/index.d.ts
index 3874c88ec4a69f91057154bc7c5247e52d45fb6a..ad335d4fcd46d6a71e5acdaef1457a01bcd5a13e 100644
--- a/dist/index.d.ts
+++ b/dist/index.d.ts
@@ -3,26 +3,3 @@ import { CoinbaseWalletProvider } from "./provider/CoinbaseWalletProvider";
 export { CoinbaseWalletSDK } from "./CoinbaseWalletSDK";
 export { CoinbaseWalletProvider } from "./provider/CoinbaseWalletProvider";
 export default CoinbaseWalletSDK;
-declare global {
-    interface Window {
-        CoinbaseWalletSDK: typeof CoinbaseWalletSDK;
-        CoinbaseWalletProvider: typeof CoinbaseWalletProvider;
-        /**
-         * For CoinbaseWalletSDK, window.ethereum is `CoinbaseWalletProvider`
-         */
-        ethereum?: any;
-        coinbaseWalletExtension?: CoinbaseWalletProvider;
-        /**
-         * @deprecated Legacy API
-         */
-        WalletLink: typeof CoinbaseWalletSDK;
-        /**
-         * @deprecated Legacy API
-         */
-        WalletLinkProvider: typeof CoinbaseWalletProvider;
-        /**
-         * @deprecated Legacy API
-         */
-        walletLinkExtension?: CoinbaseWalletProvider;
-    }
-}
diff --git a/package.json b/package.json
index 686f4e2e9d2a0a11df39bb85e472cc45777bacca..057fa691c8549159fc08e1cef9e84f601fe62ee9 100644
--- a/package.json
+++ b/package.json
@@ -19,6 +19,7 @@
   ],
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
+  "sideEffects": false,
   "repository": "https://github.com/coinbase/coinbase-wallet-sdk.git",
   "author": "Coinbase, Inc.",
   "license": "Apache-2.0",