diff --git a/dist/index.d.ts b/dist/index.d.ts
index 91e621bca4d12589af0a4b37fb24f397cf9b682f..ad335d4fcd46d6a71e5acdaef1457a01bcd5a13e 100644
--- a/dist/index.d.ts
+++ b/dist/index.d.ts
@@ -3,26 +3,3 @@ import { CoinbaseWalletProvider } from "./provider/CoinbaseWalletProvider";
 export { CoinbaseWalletSDK } from "./CoinbaseWalletSDK";
 export { CoinbaseWalletProvider } from "./provider/CoinbaseWalletProvider";
 export default CoinbaseWalletSDK;
-declare global {
-    interface Window {
-        CoinbaseWalletSDK: typeof CoinbaseWalletSDK;
-        CoinbaseWalletProvider: typeof CoinbaseWalletProvider;
-        /**
-         * For CoinbaseWalletSDK, window.ethereum is `CoinbaseWalletProvider`
-         */
-        ethereum?: unknown;
-        coinbaseWalletExtension?: CoinbaseWalletProvider;
-        /**
-         * @deprecated Legacy API
-         */
-        WalletLink: typeof CoinbaseWalletSDK;
-        /**
-         * @deprecated Legacy API
-         */
-        WalletLinkProvider: typeof CoinbaseWalletProvider;
-        /**
-         * @deprecated Legacy API
-         */
-        walletLinkExtension?: CoinbaseWalletProvider;
-    }
-}
diff --git a/package.json b/package.json
index a32df713765097bde5c670c7bc261b589282cdff..f7c3d98f762342c71b327816d8b05f20ab55dc7d 100644
--- a/package.json
+++ b/package.json
@@ -41,7 +41,7 @@
     "fix:prettier": "prettier . --write",
     "release": "./scripts/release.sh"
   },
-
+  "sideEffects": false,
   "dependencies": {
     "@metamask/safe-event-emitter": "2.0.0",
     "@solana/web3.js": "^1.70.1",