diff --git a/build/lib/index.d.ts b/build/lib/index.d.ts
index eccb8202eecfd30661e4e71c7cb279a8f829e704..9fc07bd19461cd0f4e2a8e6643bd01345d2adba2 100644
--- a/build/lib/index.d.ts
+++ b/build/lib/index.d.ts
@@ -11,10 +11,15 @@ export { focusManager } from './focusManager';
 export { onlineManager } from './onlineManager';
 export { hashKey, replaceEqualDeep, isServer, matchQuery, keepPreviousData, } from './utils';
 export type { MutationFilters, QueryFilters, Updater } from './utils';
-export { isCancelledError } from './retryer';
+export {
+  Retryer,
+  RetryValue,
+  RetryDelayValue,
+  isCancelledError,
+} from './retryer';
 export { dehydrate, hydrate, defaultShouldDehydrateQuery, defaultShouldDehydrateMutation, } from './hydration';
 export * from './types';
-export type { Query, QueryState } from './query';
+export type { Query, QueryState, QueryBehavior } from './query';
 export type { Mutation, MutationState } from './mutation';
 export type { DehydrateOptions, DehydratedState, HydrateOptions, } from './hydration';
 //# sourceMappingURL=index.d.ts.map