diff --git a/_lib/server.js b/_lib/server.js
index 16fc552b33e575f40bfee99a38e44acc29587271..b249504c631d28279786392bd6c00d189e1e178f 100644
--- a/_lib/server.js
+++ b/_lib/server.js
@@ -42,6 +42,7 @@ export function createServer(parameters) {
                 return;
             }
             const { id, path } = extractPath(url);
+            response.setHeader("Access-Control-Allow-Origin", "*");
             if (typeof id === 'number') {
                 if (path === '/') {
                     const { host, port } = pool.get(id) || (await pool.start(id));
