diff --git a/dist/client/theme-default/without-fonts.js b/dist/client/theme-default/without-fonts.js
index 2b760aa398b7ac1881551ab146d7d308ef2277a3..a2940ca0b99b7673d8914e4bf332f57c588e8f46 100644
--- a/dist/client/theme-default/without-fonts.js
+++ b/dist/client/theme-default/without-fonts.js
@@ -11,10 +11,12 @@ import Layout from './Layout.vue';
 // Note: if we add more optional components here, i.e. components that are not
 // used in the theme by default unless the user imports them, make sure to update
 // the `lazyDefaultThemeComponentsRE` regex in src/node/build/bundle.ts.
+export { default as VPButton } from './components/VPButton.vue';
 export { default as VPHomeHero } from './components/VPHomeHero.vue';
 export { default as VPHomeFeatures } from './components/VPHomeFeatures.vue';
 export { default as VPHomeSponsors } from './components/VPHomeSponsors.vue';
 export { default as VPDocAsideSponsors } from './components/VPDocAsideSponsors.vue';
+export { default as VPSponsors } from './components/VPSponsors.vue';
 export { default as VPTeamPage } from './components/VPTeamPage.vue';
 export { default as VPTeamPageTitle } from './components/VPTeamPageTitle.vue';
 export { default as VPTeamPageSection } from './components/VPTeamPageSection.vue';
diff --git a/theme.d.ts b/theme.d.ts
index 4b1076b5fbda893eb6f4a5df6a40c15d1bb379ba..177f5a6aa9602e0eeae954137bcce1ac0080ddef 100644
--- a/theme.d.ts
+++ b/theme.d.ts
@@ -3,10 +3,12 @@ import type { DefineComponent } from 'vue'
 import { EnhanceAppContext } from './dist/client/index.js'
 
 // TODO: add props for these
+export const VPButton: DefineComponent
 export const VPHomeHero: DefineComponent
 export const VPHomeFeatures: DefineComponent
 export const VPHomeSponsors: DefineComponent
 export const VPDocAsideSponsors: DefineComponent
+export const VPSponsors: DefineComponent
 export const VPTeamPage: DefineComponent
 export const VPTeamPageTitle: DefineComponent
 export const VPTeamPageSection: DefineComponent