diff --git a/dist/compiler.js b/dist/compiler.js
index b1cf495c3bc4ee202d4d833911773ecd26fb6bc1..bfbb10e792b3e13b22b84a0839f1c9387ecd34fb 100644
--- a/dist/compiler.js
+++ b/dist/compiler.js
@@ -19,6 +19,7 @@ var fse = require('fs-extra');
 var debounce = require('lodash.debounce');
 var chokidar = require('chokidar');
 var nodeModulesPolyfill = require('@esbuild-plugins/node-modules-polyfill');
+var nodeGlobalsPolyfill = require('@esbuild-plugins/node-globals-polyfill');
 var esbuildPluginPnp = require('@yarnpkg/esbuild-plugin-pnp');
 var build$1 = require('./build.js');
 var config = require('./config.js');
@@ -309,7 +310,7 @@ async function createBrowserBuild(config, options) {
     entryPoints[id] = config.routes[id].file + "?browser";
   }
 
-  let plugins = [urlImportsPlugin.urlImportsPlugin(), mdx.mdxPlugin(config), browserRouteModulesPlugin.browserRouteModulesPlugin(config, /\?browser$/), emptyModulesPlugin.emptyModulesPlugin(config, /\.server(\.[jt]sx?)?$/), nodeModulesPolyfill.NodeModulesPolyfillPlugin(), esbuildPluginPnp.pnpPlugin()];
+  let plugins = [urlImportsPlugin.urlImportsPlugin(), mdx.mdxPlugin(config), browserRouteModulesPlugin.browserRouteModulesPlugin(config, /\?browser$/), emptyModulesPlugin.emptyModulesPlugin(config, /\.server(\.[jt]sx?)?$/), nodeModulesPolyfill.NodeModulesPolyfillPlugin(), nodeGlobalsPolyfill.NodeGlobalsPolyfillPlugin(), esbuildPluginPnp.pnpPlugin()];
   return esbuild__namespace.build({
     entryPoints,
     outdir: config.assetsBuildDirectory,