diff --git a/node_modules/@remix-run/dev/dist/compiler.js b/node_modules/@remix-run/dev/dist/compiler.js
index d02216c..9734590 100644
--- a/node_modules/@remix-run/dev/dist/compiler.js
+++ b/node_modules/@remix-run/dev/dist/compiler.js
@@ -19,6 +19,7 @@ var fse = require('fs-extra');
 var debounce = require('lodash.debounce');
 var chokidar = require('chokidar');
 var nodeModulesPolyfill = require('@esbuild-plugins/node-modules-polyfill');
+var nodeGlobalsPolyfill = require('@esbuild-plugins/node-globals-polyfill');
 var esbuildPluginPnp = require('@yarnpkg/esbuild-plugin-pnp');
 var build$1 = require('./build.js');
 var config = require('./config.js');
@@ -294,9 +295,9 @@ async function createBrowserBuild(config, options) {
   let externals = module$1.builtinModules.filter(mod => !dependencies$1.includes(mod));
   let fakeBuiltins = module$1.builtinModules.filter(mod => dependencies$1.includes(mod));
 
-  if (fakeBuiltins.length > 0) {
-    throw new Error(`It appears you're using a module that is built in to node, but you installed it as a dependency which could cause problems. Please remove ${fakeBuiltins.join(", ")} before continuing.`);
-  }
+  // if (fakeBuiltins.length > 0) {
+  //   throw new Error(`It appears you're using a module that is built in to node, but you installed it as a dependency which could cause problems. Please remove ${fakeBuiltins.join(", ")} before continuing.`);
+  // }
 
   let entryPoints = {
     "entry.client": path__namespace.resolve(config.appDirectory, config.entryClientFile)
@@ -310,7 +311,7 @@ async function createBrowserBuild(config, options) {
   }
 
   let plugins = [urlImportsPlugin.urlImportsPlugin(), mdx.mdxPlugin(config), browserRouteModulesPlugin.browserRouteModulesPlugin(config, /\?browser$/), emptyModulesPlugin.emptyModulesPlugin(config, /\.server(\.[jt]sx?)?$/), // Must be placed before NodeModulesPolyfillPlugin, so yarn can resolve polyfills correctly
-  esbuildPluginPnp.pnpPlugin(), nodeModulesPolyfill.NodeModulesPolyfillPlugin()];
+  esbuildPluginPnp.pnpPlugin(), nodeModulesPolyfill.NodeModulesPolyfillPlugin(), nodeGlobalsPolyfill.NodeGlobalsPolyfillPlugin()];
   return esbuild__namespace.build({
     entryPoints,
     outdir: config.assetsBuildDirectory,
